pkg/action/yaml.def.go

```diff
@@ -284,9 +284,15 @@ type Option struct {
 	Type        jsonschema.Type `yaml:"type"`
 	Default     interface{}     `yaml:"default"`
 	Required    bool            `yaml:"required"` // @todo that conflicts with json schema object definition
+	Process     process         `yaml:"process"`
```

Comment:
Can we have multiple processors to the value? I think it's a good idea to implement it now as array.